base/src/com/google/idea/blaze/base/qsync/BazelAppInspectorBuilder.java

```diff
@@ -15,8 +15,6 @@
  */
 package com.google.idea.blaze.base.qsync;
 
-import static com.google.common.collect.ImmutableList.toImmutableList;
-
```

Comment:
To reduce merge conflicts I would advocate for prioritizing AOSP compatibility rather than cleaning up unused imports for files which are not actively maintained by us.